Tests/Fluent.Tests/Compute/VirtualMachineScaleSetTests.cs

```diff
@@ -681,13 +681,15 @@ public void CanEnableMSIWithoutRoleAssignment()
                         .Create();
 
                     var authenticatedClient = TestHelper.CreateAuthenticatedClient();
-                    //
-                    IServicePrincipal servicePrincipal = authenticatedClient
-                            .ServicePrincipals
-                            .GetById(virtualMachineScaleSet.SystemAssignedManagedServiceIdentityPrincipalId);
+                    // NOTE: Comment out since the below code need external tennat.
```

Comment:
same as above : 
Can you add a Todo instead of note, and add a GitHub issue to enable the following code once the subscription we use to record these tests supports the functionality?

Reply:
Issue https://github.com/Azure/azure-libraries-for-net/issues/739 log for this and added the TODO.